### DISCUSSION TITLE
I want to show the rendering progress of BIM or RealityModel as a percentage.

### DISCUSSION THREAD

**Discussion by Myzik-Dev** [iTwin/itwinjs-core#1457] [Q&A] [answered]
I want to show the rendering progress of BIM or RealityModel as a percentage. 
Is there an API that can implement this feature?

**Comment by pmconne** [comment, marked as answer]
[Here's an extremely simple example](https://github.com/imodeljs/imodeljs/blob/master/test-apps/display-test-app/src/frontend/TileLoadIndicator.ts) of a progress bar showing the ratio of number of loaded tiles to total number of requested tiles.

**Comment by pmconne** [comment]
Here is a [React control](https://github.com/imodeljs/imodeljs/blob/master/ui/framework/src/ui-framework/statusfields/tileloading/TileLoadingIndicator.tsx) that might do what you want.

**Reply by Myzik-Dev** [reply]
It works well in the way you told me.
Thank you for your reply. :)
